### DISCUSSION TITLE
Help Provision: Tab with content (just in case)

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1260] [Show and tell]
Hi there

![StretchedTabs](https://user-images.githubusercontent.com/5799306/194730424-4049525f-6322-4fbf-a4bc-87dbabe1cedb.gif)

This "play" is for anyone who might need help with this. I was able to have this working after looking at the source code on this repo and it just meets my needs. The code is just for demo, needs cleanup.

https://play.tailwindcss.com/RywGznSO8m

Notes:

The "tabs4tabs" holds on the tab names. There is an extra added hidden tab, so that the last tab is not having a "block" edge.
The "tabs4content" hold the contents for each of the tabs.

When a tab is Active...
Add class > [--tab-bg:hsl(var(--b2))] & tab-active
Remove class > [--tab-border-color:transparent] & hidden
If tab position = 1, then tabs4content.RemoveClass("rounded-tl-box") else tabs4content.AddClass("rounded-tl-box")

For all In-Active Tabs - execute reverse of above.

**Comment by saadeghi** [comment]
Thank you for the example!  
Moving this to the Discussions.
